demos/python_demos/object_detection_retinaface_demo/postprocessor.py

```diff
@@ -0,0 +1,253 @@
+            overlap = np.divide(intersection, union, out=np.zeros_like(intersection, dtype=float), where=union != 0)
+
+            order = order[np.where(overlap <= thresh)[0] + 1] # pylint: disable=W0143
+
+        return keep
+
+    def process_output(self, raw_output, scale_x, scale_y, face_prob_threshold):
+        bboxes_outputs = [raw_output[name][0] for name in raw_output if re.search('.bbox.', name)]
+        bboxes_outputs.sort(key=lambda x: x.shape[1])
+
+        scores_outputs = [raw_output[name][0] for name in raw_output if re.search('.cls.', name)]
+        scores_outputs.sort(key=lambda x: x.shape[1])
+
+        landmarks_outputs = [raw_output[name][0] for name in raw_output if re.search('.landmark.', name)]
+        landmarks_outputs.sort(key=lambda x: x.shape[1])
+        if self._detect_masks:
+            type_scores_outputs = [raw_output[name][0] for name in raw_output if re.search('.type.', name)]
+            type_scores_outputs.sort(key=lambda x: x.shape[1])
+
+        proposals_list = []
+        scores_list = []
+        landmarks_list = []
+        mask_scores_list = []
+        for idx, s in enumerate(self._features_stride_fpn):
+            anchor_num = self._num_anchors[s]
+            scores = self._get_scores(scores_outputs[idx], anchor_num)
+            bbox_deltas = bboxes_outputs[idx]
+            height, width = bbox_deltas.shape[1], bbox_deltas.shape[2]
+            anchors_fpn = self._anchors_fpn[s]
+            anchors = self.anchors_plane(height, width, int(s), anchors_fpn)
+            anchors = anchors.reshape((height * width * anchor_num, 4))
+            proposals = self._get_proposals(bbox_deltas, anchor_num, anchors)
+            threshold_mask = scores >= face_prob_threshold
+            proposals, scores = proposals[threshold_mask, :], scores[threshold_mask]
+            if scores.size != 0:
+                x_mins, y_mins, x_maxs, y_maxs = proposals.T
+                keep = self.nms(x_mins, y_mins, x_maxs, y_maxs, scores, 0.5)
+                proposals_list.extend(proposals[keep])
+                scores_list.extend(scores[keep])
+                landmarks = self._get_landmarks(landmarks_outputs[idx], anchor_num, anchors)[threshold_mask, :]
+                landmarks_list.extend(landmarks[keep, :])
+                if self._detect_masks:
+                    mask_scores_list.extend(self._get_mask_scores(type_scores_outputs[idx],
+                        anchor_num)[threshold_mask][keep])
+        output = namedtuple('detection', 'face_detection, landmarks_regression, mask_detection')
+        if len(scores_list) != 0:
+            scores = np.reshape(scores_list, -1)
+            mask_scores = np.reshape(mask_scores_list, -1)
+            x_mins, y_mins, x_maxs, y_maxs = np.array(proposals_list).T # pylint: disable=E0633
+            detections = [scores, x_mins / scale_x, y_mins / scale_y, x_maxs / scale_x, y_maxs / scale_y]
+
+            output.face_detection = detections
```

Comment:
You're doing something strange here. `output` is a _type_. You should create an instance of this type.